pkg/clients/logging/logger.go

```diff
@@ -113,39 +148,36 @@ func verifyLogDirectory(path string) {
 	}
 }
 
+// SetLogLevel sets minimum severity log level
+func (lc EdgeXLogger) SetLogLevel(logLevel string) {
+	if IsValidLogLevel(logLevel) == true {
+		*lc.logLevel = logLevel
```

Comment:
@jduranf Is it because the value set here will be lost when the method exits if logLevel isn't a pointer? I assume you tried it?

Reply:
Yes, if it's not a pointer the value is lost. I tried it and I spent lot of time before to realize I should use a pointer.